doc/sphinx/docs/python/tutorials/using_population.rst

```diff
@@ -91,6 +91,15 @@ length.
 .. note:: Decision vectors that are outside of the box bounds are allowed to be
           pushed back into a population
 
+The snippet above will trigger fitness function evaluations as the decision vector is always associated to a fitness vector in a pop:func:`~pygmo.core.population`.
```

Comment:
``pygmo.core.population`` is a ``:class:``, not a ``:func:``, and it looks like a ``pop`` slipped in at the beginning.